pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation_util.go

```diff
@@ -50,6 +50,12 @@ func getLibListFromDeploymentAnnotations(store workloadmeta.Component, deploymen
 	var libList []libInfo
 	for container, languages := range deployment.InjectableLanguages {
 		for lang := range languages {
+			// There's a mismatch between language detection and auto-instrumentation.
+			// The Node language is a js lib.
+			if lang == "node" {
```

Comment:
Found this in testing where if lang-detected "node" we would try to inject "node", but we don't have that in the webhook: 

https://github.com/DataDog/datadog-agent/blob/fe13deade9ef6cf2444c8c5e5284c7ef89f6794d/pkg/clusteragent/admission/mutate/autoinstrumentation/language_versions.go#L18-L25

Reply:
From taking a look at the auto-inject code, I think the name that telemetry expects is `nodejs`. Should we be consistent?